src/plotting.jl

```diff
@@ -154,12 +158,19 @@ function plot_binary_discrimination_calibration_curves!(subfig::FigurePosition,
     set_from_kw!(scatter_theme, :makersize, kw, 5)
     set_from_kw!(scatter_theme, :marker, kw, :rect)
 
-    per_expert = get_theme(subfig, :BinaryDiscriminationCalibrationCurves, :PerExpert; solid_color=:darkgrey,
-                           color=nothing)
-    set_from_kw!(per_expert, :linewidth, kw, 2)
-    ax = series_plot!(subfig, per_expert_calibration_curves, nothing; legend=nothing,
-                      title="Detection calibration", xlabel="Expert agreement rate",
-                      ylabel="Predicted positive probability", scatter=scatter_theme, per_expert...)
+    if ismissing(per_expert_calibration_curves)
+        ax = Axis(subfig; title="Detection calibration", xlabel="Expert agreement rate",
+                  ylabel="Predicted positive probability", legend=nothing)
```

Comment:
CI doesn't like the `legend` argument here, apparently:
```suggestion
        ax = Axis(subfig; title="Detection calibration", xlabel="Expert agreement rate",
                  ylabel="Predicted positive probability")
```

Reply:
Yeah that's Plots.jl speak ;)